### PR TITLE
[367] Updated overview page in line with prototype design

### PR DESCRIPTION
**Pull request by avinhurry**
### Context
Updated the overview page so that it is in line with the prototype

### Changes proposed in this pull request
Updated content and structure of the page.

### Guidance to review

The changes proposed in this PR should match the design in the prototype here: https://register-prototype.herokuapp.com/new-record/overview

### CODE REVIEW

**Comment by avinhurry** [comment]
[[S] Update task list (production) with new structure (prototype)](https://trello.com/c/1qLaXTiO/367-s-update-task-list-production-with-new-structure-prototype)

File: app/views/trainees/show.html.erb [hunk 1]
```diff
@@ -9,19 +9,28 @@
...
 <h1 class="govuk-heading-l">
-  <span class="govuk-caption-l">Add a trainee</span>
-  Overview
+  Add a trainee
 </h1>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds-from-desktop">
+
+<h2 class="govuk-heading-m">Route into teaching</h2>
```
Comment (aliuk2012) [inline app/views/trainees/show.html.erb:18]: Indenting issue

**Review by aliuk2012 — COMMENTED** [review]
minor issue with indenting

File: app/views/trainees/show.html.erb (hunk 1 above)
Comment (avinhurry) [reply app/views/trainees/show.html.erb]: thanks, updated

**Review by aliuk2012 — APPROVED** [review]

**Review by SamJamCul — APPROVED** [review]

**Pull request merged**